webapp/packages/core-blocks/src/FormControls/Combobox.tsx

```diff
@@ -270,71 +300,26 @@ export const Combobox: ComboboxType = observer(function Combobox({
         </FieldLabel>
       )}
       <div className={s(styles, { inputBox: true })}>
-        <input className={s(styles, { validationInput: true })} value={inputValue} required={rest.required} readOnly />
-        {(icon || loading) && (
-          <div className={s(styles, { inputIcon: true })}>
-            {loading ? (
-              <Loader small fullSize />
-            ) : typeof icon === 'string' ? (
-              <IconOrImage icon={icon} className={s(styles, { iconOrImage: true })} />
-            ) : (
-              icon
-            )}
-          </div>
-        )}
-        <input
-          ref={setInputRef}
-          required={rest.required}
-          autoComplete="off"
+        <SelectField
+          items={filteredItems}
+          value={value}
+          itemValue={itemValue}
+          itemRender={itemRender}
+          itemDisabled={itemDisabled}
           name={name}
-          title={title}
-          value={inputValue}
-          disabled={disabled || hideMenu}
-          readOnly={readOnly || select}
-          data-focus={focus}
-          data-select={select}
-          className={s(styles, { input: true, select, focus })}
-          onChange={handleChange}
-          onKeyDown={handleKeyDown}
-          onClick={handleClick}
+          disabled={disabled || readOnly}
+          required={rest.required}
+          className={s(styles, { field: true, inline }, className, styles['selectField'])}
+          noItemsPlaceholder={translate('combobox_no_results_placeholder')}
+          selectedRender={selectedRender}
+          arrowIcon={<Icon name="arrow" viewBox="0 0 16 16" className={styles['icon']} />}
+          store={menu}
+          popoverProps={{
+            autoFocusOnShow: !searchable,
```

Comment:
Probably it's better to expose only the `autoFocusOnShow` property instead of the `popoverProps`